packages/lesswrong/lib/collections/posts/constants.ts

```diff
@@ -31,3 +35,16 @@ export const postStatusLabels = [
     label: 'deleted'
   }
 ];
+
+const amaTagIdSetting = new DatabasePublicSetting<string | null>('amaTagId', null)
+const openThreadTagIdSetting = new DatabasePublicSetting<string | null>('openThreadTagId', null)
+const startHerePostIdSetting = new DatabasePublicSetting<string | null>('startHerePostId', null)
+
+export const postTagIcons = new Map([
```

Comment:
I kinda prefer objects here? Idk. Probably objects are more consistent with the rest of the codebase, but if you would prefer to use maps, it's probably fine. I hesitate to mention because it's fairly nit-pick-y, but I'll err on over-communicating for now.

Reply:
I appreciate erring toward over-communicating! And agreed, my default preference is for objects over maps as well. Here the keys themselves are `DatabasePublicSetting` objects (since we can't call `.get()` yet), and JS objects store keys as strings. I don't immediately know of a workaround that doesn't use something like a `Map` (other than defining the objects in the render method of `PostsTitle`, which itself seemed not ideal to me), but if you know of a way to make that work I'd be happy to switch!